src/strategies/php.ts

```diff
@@ -67,6 +68,21 @@ export class PHP extends BaseStrategy {
       }),
     });
 
+    const contents = await this.github.getFileContentsOnBranch(
+      this.addPath('VERSION'),
+      this.targetBranch
+    );
+
+    // update VERSION file
+    updates.push({
+      path: this.addPath('VERSION'),
+      createIfMissing: false,
+      cachedFileContents: contents,
```

Comment:
I am not sure what this does, or if we need it. Since we already know the version, we don't really care what's written in the file. however, I noticed this is being used in other strategies, so I left it in.

Reply:
Thanks for the explanation!